soccer/gameplay/plays/testing/test_adaptive_formation_weights.py

```diff
@@ -0,0 +1,126 @@
+import main
+import play
+import behavior
+import evaluation.field
+import evaluation.passing_positioning
+import evaluation.shooting
+import evaluation.defensive_positioning
+import constants
+import robocup
+import math
+import enum
+
+
+class TestAdaptiveFormationWeights(play.Play):
+    class State(enum.Enum):
+        # Draws 2D probabilty plot onto the field
+        # Red is higher probability
+        # Blue is lower
+        testPointCoeff = 1
+
+        # Plots all the lines from bestPass
+        # Red is higher score
+        testBestPass = 2
+
+    def __init__(self):
+        super().__init__(continuous=True)
+        self.add_state(TestAdaptiveFormationWeights.State.testPointCoeff,
+                       behavior.Behavior.State.running)
+        self.add_state(TestAdaptiveFormationWeights.State.testBestPass,
+                       behavior.Behavior.State.running)
+
+        # Enable which portion we want to test
+        mode = 1
```

Comment:
Hmm, we should probably figure out how to add that (if qt will let us), we'll switch that over when that happens I guess...